xstate-chatbot/nodejs/src/machine/messages/gis-details.js

```diff
@@ -0,0 +1,332 @@
+const messages = {
+  gisInfo: {
+    prompt: {
+      preamble: {
+        en_IN: 'What would you like to do?',
+        pa_IN: 'ਤੁਸੀਂ ਕੀ ਜਾਣਨਾ ਚਾਹੋਗੇ?',
+        hi_IN: 'आप क्या जानना चाहेंगे?',
+      },
+      options: {
+        list: ['addNewProperty', 'validateExistingProperty'],
+        messageBundle: {
+          addNewProperty: {
+            en_IN: 'Add new property',
+            pa_IN: 'ਨਵੀਂ ਜਾਇਦਾਦ ਸ਼ਾਮਲ ਕਰੋ',
+            hi_IN: 'नई संपत्ति जोड़ें',
+          },
+          validateExistingProperty: {
+            en_IN: 'Validate existing property',
+            pa_IN: 'ਮੌਜੂਦਾ ਸੰਪਤੀ ਨੂੰ ਪ੍ਰਮਾਣਿਤ ਕਰੋ',
+            hi_IN: 'मौजूदा संपत्ति सत्यापित करें',
+          },
+        },
+      },
+    },
+  },
+  propertyUseList: {
+    prompt: {
+      options: {
+        list: ['residential', 'commerical', 'residentialcommerical', 'industrial'],
+        messageBundle: {
+          residential: {
+            en_IN: 'Residential',
+            pa_IN: 'ਰਿਹਾਇਸ਼ੀ',
+            hi_IN: 'आवासीय',
+          },
+          commerical: {
+            en_IN: 'Commercial',
+            pa_IN: 'ਵਪਾਰਕ',
+            hi_IN: 'व्यावसायिक',
+          },
+          residentialcommerical: {
+            en_IN: 'Residential-Commercial',
+            pa_IN: 'ਰਿਹਾਇਸ਼ੀ-ਵਪਾਰਕ',
+            hi_IN: 'आवासीय वाणिज्यिक',
+          },
+          industrial: {
+            en_IN: 'Industrial',
+            pa_IN: 'ਉਦਯੋਗਿਕਰੋ',
+            hi_IN: 'औद्योगिक',
+          },
+        },
+      },
+    },
+  },
+  updateExistingPropertyMemu: {
+    prompt: {
+      prompt: {
+        preamble: {
+          en_IN: '<b>What would you like to update? </b>',
```

Comment:
I think, this will not make the text bold. In WhatsApp text is surrounded by * to make it bold.

Reply:
committed after removing <b>